docs/compatibility.md

```diff
@@ -1,6 +1,6 @@
 # Requests Compatibility Guide
 
-HTTPX aims to be compatible with the `requests` API wherever possible.
+HTTPX aims to be compatible with the `requests` API where we feel it makes sense.
```

Comment:
How about

```suggestion
HTTPX aims to be broadly compatible with the `requests` API.
```

I feel like the current wording creates the impression that HTTPX tries to be compatible unless that's not possible at all.

Reply:
"Broadly compatible" is spot-on, yup.